settings.lua

```diff
@@ -2494,6 +2494,14 @@ function RidgeMarkersAutomatic:isDisabled()
 	return self.vehicle.cp.driver and not AIDriverUtil.hasAIImplementWithSpecialization(self.vehicle, RidgeMarker)
 end
 
+---@class KeepCurrentSteering : BooleanSetting
+KeepCurrentSteering = CpObject(BooleanSetting)
+function KeepCurrentSteering:init(vehicle)
+	BooleanSetting.init(self, 'KeepCurrentSteering', 'COURSEPLAY_KEEP_CURRENT_STEERING',
+			'COURSEPLAY_YES_NO_KEEP_CURRENT_STEERING', vehicle)
```

Comment:
The second text is for the tooltip in the dialogs, so ideally should be called COURSEPLAY_KEEP_CURRENT_STEERING_TOOLTIP and the texts in the translation file should contain an explanation, for example "Courseplay won't not change the current crab steering setting of the vehicle."

Reply:
Done on next commit.